millicore.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"os/exec"
+	"path/filepath"
+)
+
+func GetMillicoreConfigTasks(tasks chan<- Task, runner Runner, projects []string, resourceFactory ResourceMatchFactory) {
+	for _, p := range projects {
+		pods, err := resourceFactory(p, "pod", "millicore")
+		if err != nil {
+			tasks <- NewError(err)
```

Comment:
I don't think that should be an error... we don't expect every project to have a "millicore" pod. Similar story with Nagios.